src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -91,26 +74,19 @@ export interface ModalState {
   phoneNumber: string;
 }
 
-export interface PersonPhoneNumbers {
-  phoneNumbers: PersonPhoneNumberFragment[];
-}
-
 export interface FormValuesPerson extends PersonInvalidNumberFragment {
   newPhoneNumber: string;
   isNewPhoneNumber: boolean;
 }
 
-export interface FormValues {
-  people: FormValuesPerson[];
-}
-
 interface Props {
   accountListId: string;
 }
 
 const FixPhoneNumbers: React.FC<Props> = ({ accountListId }: Props) => {
```

Comment:
How hard would it be to move all of the mutation logic inside of `Contact` now? The parent doesn't need to know how to submit changes anymore since `Contact` always decides when to submit changes now.

I don't think we need the logic that copies data state either.